src/RequirementsQuery.ts

```diff
@@ -0,0 +1,141 @@
+import { Calculator } from 'fqm-execution';
+import { R4 } from '@ahryman40k/ts-fhir-types';
+import fs from 'fs';
+import axios from 'axios';
+import { URLSearchParams } from 'url';
+import { DRQuery, APIParams, BulkDataResponse } from './types/RequirementsQueryTypes';
+
+const headers = {
+  Accept: 'application/fhir+json',
+  Prefer: 'respond-async'
+};
+const exampleMeasureBundle = '../EXM130-7.3.000-bundle.json'; //REPLACE WITH PATH TO DESIRED MEASURE BUNDLE
+const API_URL =
+  'https://bulk-data.smarthealthit.org/eyJlcnIiOiIiLCJwYWdlIjoxMDAwMCwiZHVyIjoxMCwidGx0IjoxNSwibSI6MSwic3R1IjozLCJkZWwiOjB9/fhir';
+
+const EXAMPLE_REQUIREMENTS = [
+  {
+    type: 'Encounter',
+    codeFilter: []
+  },
+  {
+    type: 'Procedure',
+    codeFilter: []
+  },
+  {
+    type: 'Patient',
+    codeFilter: []
+  }
+];
+
+/**
+ * Function taken directly from fqm-execution. parses measure bundle into
+ * appropriate format for dataRequrements function
+ * @param filePath: path to measure bundloe on local machine
+ * @returns
+ */
+function parseBundle(filePath: string): R4.IBundle {
+  const contents = fs.readFileSync(filePath, 'utf8');
+  return JSON.parse(contents) as R4.IBundle;
+}
+
+/**
+ * extracts the data requirements and fomrats
+ * @param dataRequirements: An array of data requirements as returned from fqm-execution
+ * @returns
```

Comment:
Should think about adding the return type here, and a description of what that returned object is.